src/types/response.ts

```diff
@@ -0,0 +1,4 @@
+export interface Response<T> {
```

Comment:
`ApiResponse` instead we may have multiple type of Response. Also the file name should be `api.ts`